manim/mobject/svg/svg_mobject.py

```diff
@@ -10,7 +10,9 @@
 import string
 import warnings
 
-from xml.dom import minidom
+from xml.dom.minidom import Element as MinidomElement, parse as minidom_parse
```

Comment:
This file has gone long. How about splitting it into two?

Reply:
I'd be down. Maybe breaking it into a `svg_path.py` file (~200 loc) for the VMobjectFromPathstring object, a `style_utils.py` file for methods like `cascade_element_style`, `parse_color_string`, and `parse_style` that actually don't use the SVGMobject at all?

There are other methods like `rect_to_mobject` that also don't use the SVGMobject much at all - would those be better outside of SVGMobject? I'm open to suggestions; my experience with large libraries is really limited.

Reply:
It's your preference. For me, the file should be ideally between 300-500 lines, but it's just personal preference though.

Reply:
I think that's fair. I'll pull out the `svg_path.py` file and the `style_utils.py` file but leave the rest - that seems fine.

Reply:
done in e8f4d1e